test/test_generators.py

```diff
@@ -103,28 +284,38 @@ def test_full_generator_windows(tmpdir):
     st_window_shape, min_size=1, max_size=10, unique=True)
 
 
-def create_step_and_image_strategy(limit):
-
+def create_step_and_image_strategy(args):
```

Comment:
```suggestion
def create_step_and_image_strategy(limit, dtype):
```
is much more clear

Reply:
Doing this gives me the error:
```E       TypeError: create_step_and_image_strategy() missing 1 required positional argument: 'dtype'```

Reply:
Changed this myself. The error is because you didn't change the function call accordingly.